programs/ssl/ssl_server2.c

```diff
@@ -1791,7 +1791,7 @@ int main( int argc, char *argv[] )
             unsigned char *larger_buf;
 
             ori_len = ret;
-            extra_len = ssl_get_bytes_avail( &ssl );
+            extra_len = (int) ssl_get_bytes_avail( &ssl );
```

Comment:
@RonEld: Even if we change this to size_t we will eventually need to cast as the return type of `ssl_read()` is `int`. In any case, we will not overflow the buffer because according to the RFC 5246 the length of the TLS record's content cannot exceed 2^14.